man/systemd-cryptenroll.xml

```diff
@@ -650,6 +650,51 @@
 
   </refsect1>
 
+  <refsect1>
+    <title>Credentials</title>
+
+    <para><command>systemd-cryptenroll</command> supports the service credentials logic as implemented by
+    <varname>ImportCredential=</varname>/<varname>LoadCredential=</varname>/<varname>SetCredential=</varname>
+    (see <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry> for
+    details). The following credentials are used when passed in:</para>
+
+    <variablelist class='system-credentials'>
```

Comment:
These should be referenced from the credentials manpage too

Reply:
cryptenroll is not a service that is invoked during boot. It's a command line tool, and possibly invoked via IPC eventually. 

Hence, while I think it makes a ton of sense to support the service interface for passing in creds, I am not sure it makes sense to list these creds in the system credentials list, because why would you specify them there if there's no service that would pick them up, at least not in the service units we ship?

Reply:
well I guess it depends on what's the purpose of that manpage - I thought it was supposed to be a generic list of all credentials we support?

Reply:
the man page is called *system* creds, and lists the stuff we do ImportCredentials= for so far. for cryptenroll we currently don't do that because there simply is not service file for it...